### PR TITLE
Confirm liquality style

### PR DESCRIPTION
**Pull request by hard75**

### CODE REVIEW

File: src/components/liquality/ConfirmLiqualityTransaction.vue [hunk 1]
```diff
@@ -296,6 +257,7 @@ export default class ConfirmLiqualityTransaction extends Vue {
 
   async created() {
     this.rawTx = await this.txBuilder.getUnsignedRawTx(this.pegInTxState.normalizedTx);
+    console.log('>>>>>>>>>>', this.pegInTxState.normalizedTx);
```
Comment (josedahlquist) [inline src/components/liquality/ConfirmLiqualityTransaction.vue:260]: Please remove this console log

**Review by josedahlquist — COMMENTED** [review]
except for the console log, this lgtm

**Review by josedahlquist — APPROVED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Review by leoiovlabs — CHANGES_REQUESTED** [review]
Remove changes in package-lock.json

**Review by ronaldsg20 — APPROVED** [review]
LGTM

**Review by josedahlquist — APPROVED** [review]

**Pull request merged**